ncvoter/settings.py

```diff
@@ -74,9 +74,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'ncvoter',
-        'USER': 'ncvoter',
-        'PASSWORD': 'ncvoter',
+        'NAME': 'ncvotes',
```

Comment:
I think we should keep `NAME` as `ncvoter`, so it's the same as production, but agree with setting `USER` and `PASSWORD` to the empty string to make local development easier.

Reply:
was actually intended only for local, i think we need to keep user/pw for production which connects to that RDS instance. I'll back this out and create a local setting for myself

Reply:
I believe prod settings are in a separate file (`prod_settings.py`). But yes, we can set up local settings files as an alternative.